eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/kzg/KzgTestExecutor.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Consensys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.reference.phase0.kzg;
+
+import static tech.pegasys.teku.ethtests.finder.KzgTestFinder.KZG_DATA_FILE;
+
+import java.io.IOException;
+import tech.pegasys.teku.ethtests.finder.TestDefinition;
+import tech.pegasys.teku.kzg.KZG;
+import tech.pegasys.teku.kzg.ckzg4844.CKZG4844;
+import tech.pegasys.teku.networks.Eth2NetworkConfiguration;
+import tech.pegasys.teku.reference.TestDataUtils;
+import tech.pegasys.teku.reference.TestExecutor;
+import tech.pegasys.teku.spec.config.SpecConfigDeneb;
+
+public abstract class KzgTestExecutor implements TestExecutor {
+  // We don't support config reloading and all tests are currently for mainnet
```

Comment:
Is this something we want to support in the future? If it is, it would be good to have a ticket for it linked here.

Reply:
1. We may need it not only here, Stefan is already following it for another Kzg PR
2. There are no reference tests other than for `mainnet` at the moment